text/0021-internal-data-monitoring.md

```diff
@@ -0,0 +1,222 @@
+
+Project 1 with project applications would use the [gspread API](https://docs.gspread.org/en/latest/user-guide.html)
+to obtain the data from the spreadsheets and parse through/organize the
+necessary statistics. Then, the post-processed organized data would be stored
+into Firebase, where we would use [React-Vis](https://uber.github.io/react-vis/) to display the data
+and come up with concrete conclusions and inferences.
+
+### Project 2: Social Media Data Vis
+The second project would be a program that monitors how others interact
+with Creative Labs. This would be primarily to understand the correlation
+between social media post interactions (ads, likes, shares) and event
+attendance. 
+
+Project 2 with social media monitoring would use the [Facebook API](https://developers.facebook.com/docs/graph-api/reference/page/)
+and the [Instagram API](https://developers.facebook.com/docs/instagram-api/) to obtain the neccesary datapoints. We would
+need to map each post and advertisement run to a specific event hosted by
+Creative Labs and quantify the effect or potential correlation. This
+mapping would be done by giving each event in Creative Labs a certain
+time frame and checking the timestamp of each post or advertisement run
+to see which fall into that time frame.
+
+### Project 3: Website Data Vis
+The third project would dive into where (and at what frequency) users
+are navigating towards on our main website.
+
+Project 3 with monitoring for the main website would require integration with
+the implementation of the site in the future.
+
+## What are we launching today?
+
+The most important part of this project is allowing board members to easily
+access more internal data and its visualization. A barebones version would
+be successfully creating a web app that allows users to authenticate themselves
+and see a general overview of the project data. The first main add on to this
+version would be an interface that allows the user to select a certain time
+frame to view, or other various options such as looking solely at iOS or game
+development projects and their statistics. The second main addition to the
+barebones version would be a monitoring platform for both Creative Labs social
+media interactions and main website traffic.
+
+## FAQ
+
+> Working backwards, what are some questions a user, developer, designer, marketer
+> have regarding the project? Answering these questions ahead of time will make
+> the development process smoother and better for everyone involved.
+
+## Appendix
+
+The original Watchdog project can be found [here](https://github.com/UCLA-Creative-Labs/watchdog/).
+
+For Project 1, we could use the following data tables for each project member,
+project, application cycle, and time period.
+
+Project member:
+```json
+{
+    "<email>": {
+        "name": "<name>",
+        "year": "<year>",
+        "major": "<major>",
+        "application_cycles": ["<cycle_id1>", "<cycle_id2>", ...],
```

Comment:
not sure if application_cycle is the right term anymore.. these look like applications right?